### ISSUE TITLE
Kick attacks bypass AC checks

### ISSUE THREAD

**Issue by k21971** [NetHack/NetHack#440]
Meaning a level one player (not poly'd) could theoretically kill any monster with the right resources, no matter how good that monsters AC is.

Test I did to confirm - rolled a Tourist in wizmode, wished for a set of dragon scale mail (type is irrelevant) and made the enchantment ridiculously high (in this case +70). Dropped the DSM, generated a sergeant, and let them pickup and equip the +70 DSM, giving them an effective AC of -72 (confirmed via wand of probing). Was able to kill the sergeant via kicking only - every single kick landed a hit.

I understand that a kick attack from a non-poly'd player doesn't do very much damage, but this behavior can't be intentional can it?

**Comment by copperwater** [comment]
Also known as "kicking a ghost to death" because they're hard to hit normally, but you can off them by kicking and whittling their HP down with each kick.

**Comment by amateurhourrrr** [comment]
I'm surprised that you haven't heard of this before K2

I don't think this is a problem and will be sad to see it go
other than this really niche use (kicking early game ghosts) kicking is completely pointless

**Comment by davidssmith** [comment]
It's an old mechanic, but not consistent with AC checking attacks from monsters that kick, like `u`.

**Comment by Planet-Patrick** [comment]
Kicking boots should always hit though; otherwise that niche +5 damage feature no longer seems so good.

**Comment by paxed** [comment]
Has any variant changed this?

**Comment by k21971** [comment]
> Has any variant changed this?

I don't believe so. I know none of the 3.6/3.7 based variants have changed this. I also had a look at Slash'EM, dNetHack, and GruntHack - there isn't any significant changes to kick_monster() in any of those.

**Comment by k21971** [comment]
https://github.com/k21971/EvilHack/commit/b8a11c717a7755417e0046329110855350197588 <--- fixed.
